src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
```

Comment:
Not entirely. The main challenge here is that the service worker is a result of a build plugin - it can't be separated from the build config - so we need to depend on the builder for info about where the file ends up. There might be some options for making the dependency structures more clear.